src/tests/ftest/util/ior_test_base.py

```diff
@@ -263,9 +264,12 @@ def run_ior(self, manager, processes, intercept=None, display_space=True,
             manager.working_dir.value = self.dfuse.mount_dir.value
         manager.assign_hosts(
             self.hostlist_clients, self.workdir, self.hostfile_clients_slots)
-        if self.processes_per_node:
-            processes = self.processes_per_node * (len(self.hostlist_clients))
-        manager.assign_processes(processes)
+        if self.ppn is None:
+            manager.assign_processes(processes)
+        else:
+            manager.ppn.update(self.ppn, 'mpirun.ppn')
+            manager.processes.update(None, 'mpirun.np')
+
```

Comment:
It looks like mpirun has ppn, but orterun uses pprnode: https://github.com/daos-stack/daos/blob/f022d25c47fcb96955d0c5d4e90e04037f674a9b/src/tests/ftest/util/job_manager_utils.py#L235

setUp specifies MPICH, which uses mpirun, but do we need a more generic function than directly updating ppn? E.g. assign_ppn(ppn)

Reply:
As discussed earlier in the meeting today, we always use mpich for ior/mdtest as we need to test for MPIIO along with other apis. But having said, Phil discussed his long term plan in the same meeting as per which I believe your concern will be taken care off.

Reply:
Makes sense. I think this is okay for now.